spec/controllers/users/piv_cac_login_controller_spec.rb

```diff
@@ -40,12 +40,6 @@
         it 'redirects to the error url' do
           expect(response).to redirect_to(login_piv_cac_error_url(error: 'token.bad'))
         end
-
-        it 'records unsuccessful 2fa event' do
-          expect(controller).to receive(:create_user_event).with(:sign_in_unsuccessful_2fa)
-
-          response
-        end
```

Comment:
Noting that this is being removed because it never did anything anyways. If the token is invalid, there won't be a user associated, so [`create_user_event` won't do anything](https://github.com/18F/identity-idp/blob/5a0629630f4beb068a4cb31c696ef26767d5a18c/app/services/user_event_creator.rb#L15).